app/questionnaire/questionnaire_schema.py

```diff
@@ -497,3 +497,10 @@ def _get_error_messages(self):
             messages.update(self.json["messages"])
 
         return messages
+
+    @staticmethod
+    def show_individual_response(location, schema):
+        section = schema.json.get("individual_response", {}).get(
```

Comment:
I think you've misunderstood my original comment here - the first line here that returns the section_id should be a separate method I think and shouldn't be static. I think this method name should probably change too as it doesn't describe the check. Something like `is_individual_response_interstitial` makes more sense.

Reply:
For this method, `is_first_block_in_indiividual_response` is probably more appropriate since it being an interstitial is irrelevant.

Reply:
@iwootten that first line is now a method in Questionnaire Schema class. Thanks!